src/main/java/net/education/kyivstar/Application.java

```diff
@@ -0,0 +1,34 @@
+package net.education.kyivstar;
+
+import com.github.javafaker.Faker;
+
+import java.util.Random;
+
+import static net.education.kyivstar.Service.UserType.*;
+import static net.education.kyivstar.Storage.*;
+
+public class Application {
+
+    public static void main(String[] args) {
+        Faker faker = new Faker();
+        Storage storage = new Storage();
+        Random random = new Random();
+
+        Service service = new Service(faker,storage,random);
+
+        service.populateStorage(1000);
+
+        service.createUserAndStore(STUDENT,"Alex","Burmistrenko",30);
+        service.createUserAndStore(TEACHER,"Maxim","Dzhezhelo",30);
+
+        System.out.println("Get by name method: " + service.getByName("Alex"));
```

Comment:
Use Logger
https://betterstack.com/community/guides/logging/how-to-start-logging-with-java/

Reply:
done

Reply:
Why didn't you remove sout? 

please, follow this https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53